src/utils/wallet.js

```diff
@@ -153,10 +154,31 @@ export class Wallet {
         if (this.isEmpty()) {
             throw new Error('No account.')
         }
+
+        const [ fromChain, fromDb ] = await Promise.all([
+            this.getAccount(this.accountId).state(),
+            sendJson('POST', `${ACCOUNT_HELPER_URL}/account/recoveryMethods`, {
+                accountId: this.accountId,
+                ...(await this.signatureFor(this.accountId))
+            })
+        ]);
+
+        const recoveryMethods = {}
+        if (fromDb.email) {
```

Comment:
why not return data from contract helper in already desired format?

Reply:
Done!